env/.env.firebase

```diff
@@ -1,7 +1,7 @@
 REACT_APP_FIREBASE="true"
-REACT_APP_apiKey=
-REACT_APP_authDomain=
-REACT_APP_projectId=
-REACT_APP_storageBucket=
-REACT_APP_messagingSenderId=
-REACT_APP_appId=
\ No newline at end of file
+REACT_APP_apiKey="AIzaSyB6C2nVXk__dqxdyyROFEC-ue50O_OsjTE"
+REACT_APP_authDomain="ccv-honeycomb.firebaseapp.com"
+REACT_APP_projectId="ccv-honeycomb"
+REACT_APP_storageBucket="ccv-honeycomb.appspot.com"
+REACT_APP_messagingSenderId="449496296276"
+REACT_APP_appId="1:449496296276:web:0479c82b8b068ca9e6b6c1"
```

Comment:
Should we include API key in the base repo?

Reply:
These are safe because they're the credentials for Firebase's client side API. They can be looked up on the client anyways, and the GH actions use them for deploying the preview links and things like that.

The keys for Firebase Admin are secrets but those are only used in the download script, I haven't set that account up locally and don't plan to

Reply:
Sounds good to me :)